spec.html

```diff
@@ -35302,7 +35303,7 @@ <h1>
               1. If _max_ is +&infin;, let _max2_ be +&infin;; otherwise let _max2_ be _max_ - 1.
               1. Return RepeatMatcher(_m_, _min2_, _max2_, _greedy_, _y_, _c_, _parenIndex_, _parenCount_).
             1. Let _cap_ be a copy of _x_'s _captures_ List.
-            1. [id="step-repeatmatcher-clear-captures"] For each integer _k_ such that _parenIndex_ &lt; _k_ and _k_ &le; _parenIndex_ + _parenCount_, set _cap_[_k_] to *undefined*.
+            1. [id="step-repeatmatcher-clear-captures"] For each integer _k_ in the interval from _parenIndex_ (exclusive) to _parenIndex_ + _parenCount_ (inclusive), set _cap_[_k_] to *undefined*.
```

Comment:
Ugh, gross. I realize we've defined things in a way which lets you write this but half-open intervals where the closed half is the _upper_ bound are unnatural. If we must use an explicit interval here, let it be `the inclusive interval from _parenIndex_ + 1 to _parenIndex_ + _parenCount_`.

Reply:
It's just the same interval that was already being described. But if you want it changed, I'll change it.

Reply:
The problem isn't the set of numbers being described, it's the way of describing it.

Reply:
You mean the notation, right? Because they're basically the same to me, each is a different way to describe this particular interval.

Reply:
Yes, "way of describing" means the notation used, and "this particular interval" means the set of numbers. My problem is not with the set of numbers being described, it's the way of describing it.

Reply:
"way of describing" could mean either the bounds used or the notation (prose vs comparisons). I didn't understand because you were asking for a change in the bounds but the change I've made in this PR is only to the notation. So I guess those bounds are acceptable to you using the previous notation but not the new notation? To me, changing the bounds seems like an unrelated change that could warrant its own small PR.

Reply:
Okay, to me those aspects were independent. I will change the bounds.